Lab78.ipynb

```diff
@@ -0,0 +1,575 @@
+      "text/plain": [
+       "<Figure size 864x432 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "fig, axs = plt.subplots(1,1,figsize=(12,6))\n",
```

Comment:
Don't forget to add titles, descriptions, etc., for these waterfall plots and other plots. Also I think it would be useful to label the colorbar to note that the colors represent the magnitude of the visibility at each value